pico_w/wifi/dtls/certs/makecerts.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/bash
+
+if [ "${PWD##*/}" != "certs" ]; then
+    echo Run this in the certs folder
+    exit 1
+fi
+if [ -z "$DTLS_SERVER" ]; then
+    echo Define DTLS_SERVER
+    exit 1
+fi
+SERVER_NAME=$DTLS_SERVER
+
+if [ -d "$SERVER_NAME" ]; then
+    echo Run \"rm -fr $SERVER_NAME\" to regenerate these keys
+    exit 1
+fi
+mkdir $SERVER_NAME
+echo Generating keys in $PWD/$SERVER_NAME
+
+openssl genrsa -out $SERVER_NAME/ca.key 2048
```

Comment:
Maybe it's worth checking that the `openssl` command is available, and displaying an error if not? (which should obviously be done before `mkdir $SERVER_NAME`)